### PR TITLE
Shinyapps deployment

### PR DESCRIPTION
**Pull request by brucehoff**
This PR adds the GitHub workflow for deployment to ShinyApps. It also removes the use of `config.yml` for setting the application URL which is cumbersome: Deployment is much simpler if the URL can simply be passed in as an environment variable.

### CODE REVIEW

File: .github/workflows/shinyapps_deploy.yml
Comment (JessterB) [inline .github/workflows/shinyapps_deploy.yml:11]: Should we remove the 'shinyapps-and-docker' branch from here?

File: .github/workflows/shinyapps_deploy.yml
Comment (JessterB) [inline .github/workflows/shinyapps_deploy.yml:44]: @j-hendrickson-sage Note that this is set up so that we will need to start creating release branches in order to deploy to production. Let's sync up to talk about our gitflow and versioning before you try to deploy to prod.

**Review by JessterB — APPROVED** [review]
Thanks so much for all the work you put in on this Bruce, this is great! Just one little nitpick, which we can resolve on our side later if you prefer.

File: .github/workflows/shinyapps_deploy.yml
Comment (brucehoff) [reply .github/workflows/shinyapps_deploy.yml:11]: Can you remove after you merge?  Keeping it in for the moment allows me to see that the build works on my own fork.  Once you merge and then deploy, feel free to remove this line.

File: .github/workflows/shinyapps_deploy.yml
Comment (brucehoff) [reply .github/workflows/shinyapps_deploy.yml:44]: Elsewhere we deploy based on a *tag* rather than a *branch*, which is totally legit'.  You can feel free to do it either way.

Comment (j-hendrickson-sage) [reply .github/workflows/shinyapps_deploy.yml:44]: @JessterB Let's sync tomorrow; I will hold off approval until I get more time to inspect the changes.

**Pull request merged**
